{"uid":"2e0b36e513143ded","name":"test_login_form","fullName":"tests.test_login_page.TestLoginForm#test_login_form","historyId":"2e2e12a0da22bfb49860bf88dbe338c6","time":{"start":1689186454318,"stop":1689186454318,"duration":0},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55c2654e24e3 <unknown>\n#1 0x55c265211c76 <unknown>\n#2 0x55c26523ad78 <unknown>\n#3 0x55c265237029 <unknown>\n#4 0x55c265275ccc <unknown>\n#5 0x55c26527547f <unknown>\n#6 0x55c26526cde3 <unknown>\n#7 0x55c2652422dd <unknown>\n#8 0x55c26524334e <unknown>\n#9 0x55c2654a23e4 <unknown>\n#10 0x55c2654a63d7 <unknown>\n#11 0x55c2654b0b20 <unknown>\n#12 0x55c2654a7023 <unknown>\n#13 0x55c2654751aa <unknown>\n#14 0x55c2654cb6b8 <unknown>\n#15 0x55c2654cb847 <unknown>\n#16 0x55c2654db243 <unknown>\n#17 0x7f9385602b43 <unknown>","statusTrace":"E   selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n      (unknown error: DevToolsActivePort file doesn't exist)\n      (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\n    Stacktrace:\n    #0 0x55c2654e24e3 <unknown>\n    #1 0x55c265211c76 <unknown>\n    #2 0x55c26523ad78 <unknown>\n    #3 0x55c265237029 <unknown>\n    #4 0x55c265275ccc <unknown>\n    #5 0x55c26527547f <unknown>\n    #6 0x55c26526cde3 <unknown>\n    #7 0x55c2652422dd <unknown>\n    #8 0x55c26524334e <unknown>\n    #9 0x55c2654a23e4 <unknown>\n    #10 0x55c2654a63d7 <unknown>\n    #11 0x55c2654b0b20 <unknown>\n    #12 0x55c2654a7023 <unknown>\n    #13 0x55c2654751aa <unknown>\n    #14 0x55c2654cb6b8 <unknown>\n    #15 0x55c2654cb847 <unknown>\n    #16 0x55c2654db243 <unknown>\n    #17 0x7f9385602b43 <unknown>","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"browser","time":{"start":1689186454319,"stop":1689186454976,"duration":657},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55c2654e24e3 <unknown>\n#1 0x55c265211c76 <unknown>\n#2 0x55c26523ad78 <unknown>\n#3 0x55c265237029 <unknown>\n#4 0x55c265275ccc <unknown>\n#5 0x55c26527547f <unknown>\n#6 0x55c26526cde3 <unknown>\n#7 0x55c2652422dd <unknown>\n#8 0x55c26524334e <unknown>\n#9 0x55c2654a23e4 <unknown>\n#10 0x55c2654a63d7 <unknown>\n#11 0x55c2654b0b20 <unknown>\n#12 0x55c2654a7023 <unknown>\n#13 0x55c2654751aa <unknown>\n#14 0x55c2654cb6b8 <unknown>\n#15 0x55c2654cb847 <unknown>\n#16 0x55c2654db243 <unknown>\n#17 0x7f9385602b43 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/testing_foodgram_project_react/testing_foodgram_project_react/conftest.py\", line 26, in browser\n    browser = webdriver.Chrome(\n              ^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/chrome/webdriver.py\", line 84, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/chromium/webdriver.py\", line 104, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 286, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 378, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"afterStages":[],"labels":[{"name":"feature","value":"Тесты аутентификации зарегистрированного пользователя."},{"name":"story","value":"Тест аутентификации."},{"name":"tag","value":"login_test(scope='function')"},{"name":"tag","value":"login_form_test(scope='class')"},{"name":"tag","value":"run(order=2)"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login_page"},{"name":"subSuite","value":"TestLoginForm"},{"name":"host","value":"fv-az256-210"},{"name":"thread","value":"2518-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login_page"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["login_form_test(scope='class')","run(order=2)","login_test(scope='function')"]},"source":"2e0b36e513143ded.json","parameterValues":[]}